src/main/java/io/jenkins/plugins/orka/AgentTemplate.java

```diff
@@ -95,6 +95,29 @@ public AgentTemplate(String vmCredentialsId, String vm, boolean createNewVMConfi
         this.jvmOptions = jvmOptions;
```

Comment:
I suggest using the `this` expression in order to call the new ctor. You can check [here](https://github.com/jenkinsci/macstadium-orka-plugin/pull/81/files#diff-7850c74fc1980533d5c1fc322082e873dfe32d852ee420ebd2706ef960944f85R73)

Reply:
It should be the other way around :) The less specific ctor (line 77) should call more specific one.
It a nutshell - only one ctor should assign all values. The other should just call it with "this"

Reply:
Why is this resolved? This ctor is still not calling the new one